src/SlidingSyncManager.ts

```diff
@@ -109,10 +109,12 @@ export class SlidingSyncManager {
     public configure(client: MatrixClient, proxyUrl: string): SlidingSync {
         this.client = client;
         this.listIdToIndex = {};
+        DEFAULT_ROOM_SUBSCRIPTION_INFO.include_old_rooms.required_state.push(
+            [EventType.RoomMember, client.getUserId()],
+        );
         this.slidingSync = new SlidingSync(
             proxyUrl, [], DEFAULT_ROOM_SUBSCRIPTION_INFO, client, SLIDING_SYNC_TIMEOUT_MS,
         );
-        // set the space list
```

Comment:
this seemed informative - was it inaccurate?

Reply:
Seemed a bit redundant to me, but if it isn't for others I'll add it back in!